vnext/Microsoft.ReactNative.Cxx/JSValueWriter.h

```diff
@@ -23,6 +23,8 @@ template <class T, std::enable_if_t<std::is_convertible_v<T, std::string_view>,
 void WriteValue(IJSValueWriter const &writer, T const &value) noexcept;
 template <class T, std::enable_if_t<std::is_convertible_v<T, std::wstring_view>, int> = 1>
 void WriteValue(IJSValueWriter const &writer, T const &value) noexcept;
+void WriteValue(IJSValueWriter const &writer, winrt::hstring value) noexcept;
+void WriteValue(IJSValueWriter const &writer, const winrt::param::hstring value) noexcept;
```

Comment:
I wonder why is it needed: the code for wstring_view above must just work.

Reply:
this came from a comment you yourself had made in a PR :) https://github.com/microsoft/react-native-windows-samples/pull/280#discussion_r498542159

Reply:
Ah, I see the point: the wstring_view requires extra memory allocation. Never mind. :)